lib/support/cli.js

```diff
@@ -117,16 +117,26 @@ module.exports.parseCommandLine = function parseCommandLine() {
       describe: 'Choose which device to use. If you do not set it, random device will be used.',
       group: 'chrome'
     })
-    .option('chrome.dumpTraceCategoriesLog', {
+    .option('chrome.collectTraceCategoriesLog', {
       type: 'boolean',
-      describe: 'Dump Chromes traceCategories log to disk.',
+      describe: 'Collect Chromes traceCategories log to disk.',
       group: 'chrome'
     })
     .option('chrome.traceCategories', {
       describe: 'Set the trace categories.',
       type: 'string',
       group: 'chrome'
     })
+    .option('chrome.collectChromePerflog', {
```

Comment:
Now that it's in the Chrome category, I suggest we shorten it to chrome.collectPerflog.